modules/Hoot/ui/modals/exportData.js

```diff
@@ -240,7 +240,7 @@ export default class ExportData {
             .then( async resp => {
                 if (resp.data && resp.data.status !== 'cancelled') {
                     await Hoot.api.saveDataset( this.jobId, data.outputname + '.' + data.outputtype + '.zip' );
-                    // or open in josm remote control
+                    // or open in JOSM remote control
                     //Hoot.api.openDatasetInJosm( this.jobId, data.outputname + '.' + data.outputtype );
```

Comment:
We should delete these two lines, they were just a placeholder when I was testing.